packages/core-sdk/test/integration/license.test.ts

```diff
@@ -239,27 +239,39 @@ describe("License Functions", () => {
       expect(result.tokenAmount).to.be.a("bigint");
     });
 
-    it("should set licensing config", async () => {
-      const result = await client.license.setLicensingConfig({
-        ipId: ipId,
-        licenseTermsId: licenseId,
-        licenseTemplate: piLicenseTemplateAddress[aeneid],
-        licensingConfig: {
-          mintingFee: 0,
-          isSet: true,
-          licensingHook: zeroAddress,
-          hookData: "0xFcd3243590d29B131a26B1554B0b21a5B43e622e",
-          commercialRevShare: 0,
-          disabled: false,
-          expectMinimumGroupRewardShare: "1",
-          expectGroupRewardPool: zeroAddress,
-        },
-        txOptions: {
-          waitForTransaction: true,
-        },
+    describe("set and get licensing config", () => {
+      const licensingConfig: LicensingConfig = {
+        mintingFee: 0n,
+        isSet: true,
+        licensingHook: zeroAddress,
+        hookData: "0xfcd3243590d29b131a26b1554b0b21a5b43e622e",
```

Comment:
let's make this random, so that each test will set this to a different value. otherwise we are setting it to the same thing everytime, which isn't the best for the `get licensing config` test validation.